template/private-npm-package/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "<%- fullModuleName %>",
+  "description": "<%- description %>",
+  "version": "0.0.0-semantically-released",
+  "main": "lib-commonjs",
+  "module": "lib-es2015",
+  "typings": "lib-types",
```

Comment:
We've been pretty bad about this to date, but I think we should encourage using patterns like `./lib-types/index.d.ts` going forwards, to avoid potential issues in future with `exports`: https://medium.com/javascript-in-plain-english/is-promise-post-mortem-cab807f18dcc